src/Cli/dotnet/Commands/CliCommandStrings.resx

```diff
@@ -2522,4 +2522,7 @@ Proceed?</value>
     <value>Tool package download needs confirmation. Run in interactive mode or use the "--yes" command-line option to confirm.</value>
     <comment>{Locked="--yes"}</comment>
   </data>
-</root>
+  <data name="ToolInstallPackageIdMissing" xml:space="preserve">
+    <value>A package ID was not specified for tool installation.</value>
+  </data>
```

Comment:
There was a null failure path that wasn't being handled so when removing nullable disable, it was easiest to check that failure path and throw this new message if it occurs.